core/ibft.go

```diff
@@ -1055,7 +1050,7 @@ func (i *IBFT) validPC(
 	)
 
 	// Make sure there are at least Quorum (PP + P) messages
-	if len(allMessages) < int(i.backend.Quorum(i.state.getHeight())) {
+	if !i.backend.HasQuorum(i.state.getHeight(), allMessages, proto.MessageType_PREPARE) {
```

Comment:
This is where the issue lies, the PREPARE message handler in the call requires quorum - 1, when it should require quorum

Reply:
and that is the difference before and after this PR which we hope will solve the failing test.